styles/review.less

```diff
@@ -406,6 +406,10 @@
     margin-left: @component-padding;
   }
 
+  &-authorAssociationBadge {
+    margin-left: @component-padding;
```

Comment:
nit: I think the vertical alignment of the badges is a tiny bit off.  Lemmesee if I can fix it.

Reply:
you know what...it's not that the badge is vertically not aligned.  The author's avatar is aligned with the top of the text, not the middle!  The badge is fine.